app/jobs/web_of_science/author_harvest_job.rb

```diff
@@ -0,0 +1,19 @@
+module WebOfScience
+  class AuthorHarvestJob < ActiveJob::Base
+    queue_as :wos_author_harvest
+
+    # Performs an asynchronous harvest and save for a given Author
+    # @param [Author] author
+    # @param [Hash] options
+    # @return [void]
+    def perform(author, options = {})
+      return unless Settings.WOS.enabled
```

Comment:
I think the protection should be against even enqueuing a job if the setting is disabled.  No point filling up a queue just to figure out 10,000 times asynchronously that nothing should happen.  (This could block other jobs that actually might do something.  Just naming a different `queue_as` isn't sufficient to know how the backend is processing.)

Reply:
Done
- required changes in the AuthorsController
- safe guards are still required in the jobs, esp if an API endpoint is not functional, to avoid retries